test/integration/user/api/interactivity.test.js

```diff
@@ -67,6 +67,10 @@ describe('Interactivity', () => {
             it('should fire a featureClick event with a features list containing the clicked feature', done => {
                 interactivity = new carto.Interactivity(layer);
                 interactivity.on('featureClick', event => {
+                    expect(event.features[0].style.color.blendTo).toBeDefined();
+                    expect(event.features[0].style.color.reset).toBeDefined();
+                    expect(event.features[0].style.reset).toBeDefined();
+                    delete event.features[0].style;
```

Comment:
Please use this instead:
```js
expect(event.features[0]).toEqual({ id: 0, layerId: 'layer', properties: { cartodb_id: 0, style: jasmine.any(Object) } });
```

Reply:
Yes... That's better I asked off-channel about exactly this before doing this ñapa :crying_cat_face: